README.md

```diff
@@ -121,24 +122,13 @@ grid: {
 }
 ````
 
-Or, in a function:
+Using the format option allows you to change the displayed value ($10):
```

Comment:
Can we change this to "allows you to format the displayed value ($10)"? The brilliance of `format` is that it is all about *formatting*, not about *changing*.